src/julia.h

```diff
@@ -1280,6 +1280,22 @@ jl_value_t *jl_apply(jl_function_t *f, jl_value_t **args, uint32_t nargs)
     return f->fptr((jl_value_t*)f, args, nargs);
 }
 
+STATIC_INLINE
+jl_value_t *jl_apply_unspecialized(jl_function_t *meth, jl_value_t **args, uint32_t nargs)
```

Comment:
I would probably move this to gf.c; I certainly hope it won't need to be used anywhere else.